dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/rserve/RemoteDataFrameService.java

```diff
@@ -580,100 +566,101 @@ public File runDataPreprocessing(DataFile dataFile) {
                 rOutStream.write(accessObject.getVarHeader().getBytes());
             }
 
-            copyWithBuffer(is, rOutStream, 4*8192); 
-            
+            copyWithBuffer(is, rOutStream, 4 * 8192);
+
             // Rserve code starts here
-            logger.fine("wrkdir="+RSERVE_TMP_DIR);
-            
+            logger.fine("wrkdir=" + RSERVE_TMP_DIR);
+
             // Locate the R code and run it on the temp file we've just 
             // created: 
-            
+
             connection.voidEval("library(rjson)");
+            String DATAVERSE_R_PREPROCESSING = "scripts/preprocess.R";
```

Comment:
We can, but should it stay static even when it was not used outside of the class?

Reply:
Yes. If it is not used then even better. It could be private then.

I see static fields in class as some constants. Value that will never change during app execution. As opposite to non-static field that can change during time.

Because it is named in uppercase - it also suggests that is constant value